File: exercice3.c
#include <stdio.h>

int main(void)//on initialise notre fonction principale
{
    int a = 0;//on initialise notre variable a
    int b = 0;//on initialise notre variable b
    int c = 0;//on initialise notre variable b
    int max = 0;//on initialise notre variable max
    
    printf("premier chiffre : \n");//on demande le premier chiffre
    scanf("%i", &a);//on donne une valeur à la variable a
    
    printf("deuxieme chiffre : \n");//on demande le deuxieme chiffre
    scanf("%i", &b);//on donne une valeur à la variable b
    
    printf("troisieme chiffre : \n");//on demande le troisieme chiffre
    scanf("%i", &c);//on donne une valeur à la variable c
    
    if (a > b)//si la variable a est supérieur à la variable b
    {
        max = a;//on attribue la variable a à la variable max
    }
    else if (b > c)//autrement si la variable b est supérieur à la variable c 
    {
        max = b;//on attribue la variable b à la variable max
    }
    else
    {
        max = c;//on attribue la variable c à la variable max
    }
    printf("le plus grand chiffre est : %i\n", max);//afficher le plus grand chiffre
}

File: exercice3.py
a = int(input("a : "))  # on attribue à la variable a le fait que ce soit un entier et que l'on doit rentrer la valeur de a
b = int(input("b : "))  # on attribue à la variable b le fait que ce soit un entier et que l'on doit rentrer la valeur de b
c = int(input("c : "))  # on attribue à la variable c le fait que ce soit un entier et que l'on doit rentrer la valeur de c

if a < b and b < c:  # si la variable a est spérieur à la variable b et que la variable b est supérieur à la variable c
    print(f"la valeur la plus haute est {c}")  # on affiche la variable c

if b < c and c < a:  # si la variable b est spérieur à la variable c et que la variable c est supérieur à la variable a
    print(f"la valeur la plus haute est {a}")  # on affiche la variable a

if c < a and a < b:  # si la variable a est spérieur à la variable b et que la variable b est supérieur à la variable c
    print(f"la vaur la plus haute est {b}")  # on affiche la variable b

